packages/survey-creator-core/src/questionconverter.ts

```diff
@@ -12,7 +13,9 @@ export class QuestionConverter {
     className: string,
     availableTypes: Array<string> = null
   ): Array<string> {
-    var res = QuestionConverter.convertInfo[className];
+    var res = settings.allowToConvertQuestionsToAllTypes
```

Comment:
As I understand we trying to not use `var` statement anymore. We use `const` and `let` statements instead